reference/7.2/Microsoft.PowerShell.Core/About/about_ANSI_Terminals.md

```diff
@@ -83,7 +83,18 @@ The following members control how or when ANSI formatting is used:
   - **PlainText**: ANSI escape sequences are always stripped so that it is only
     plain text.
   - **Host**: The ANSI escape sequences are removed in redirected or piped
-    output.
+
+  > [!NOTE]
+  > When working with pure strings `$PSStyle.OutputRendering` does not change the
+  > output behavior of the string. For example:
+  >
+  > ```powershell
+  > $PSStyle.Foreground.Red + "abc" + $PSStyle.Reset | % { $_.Length }
+  > 12
+  > ```
```

Comment:
@sdwheeler `$PSStyle.OutputRendering` only affects `Out-String` and `Out-File` and the `Host`. It doesn't affect other commands or pipeline.